examples/connectivity/plot_mne_inverse_label_connectivity.py

```diff
@@ -49,6 +56,27 @@
                     baseline=(None, 0), reject=dict(mag=4e-12, grad=4000e-13,
                                                     eog=150e-6))
 
+###############################################################################
+# Compute inverse solutions and their connectivity
+# ------------------------------------------------
+#
+# Next, we need to compute the inverse solution for this data. This will return
+# the sources / source activity that we'll use in computing connectivity.
+#
+# Now we are ready to compute the connectivity in the alpha band. Notice
+# from the status messages how mne-python:
+#
+#  1. reads an epoch from the raw file
```

Comment:
The extra space before the 1. here makes it have the "quotation" blocking, which we don't need

https://4112-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/connectivity/plot_mne_inverse_label_connectivity.html#compute-inverse-solutions-and-their-connectivity